targets/XPlatCppSdk/source/code/include/playfab/OneDSIOSHttpPlugin.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#ifndef DISABLE_ONEDS_API
```

Comment:
Shouldn't this whole file be wrapped in:
#if defined(PLAYFAB_PLATFORM_IOS)
?

Reply:
I followed the implementation of other platforms. But, yes it will be better to be wrapped or make subdirectories. It will be good to do once later.